src/ui/EpisodesList/EpisodesList.tsx

```diff
@@ -21,6 +24,26 @@ export function EpisodesList({ className = '', episodes }: EpisodesListProps) {
 
   return (
     <div className={`${styles.container} ${className}`} ref={containerRef}>
+      {children}
+      <FeatureToggle feature="episodesFilter">
```

Comment:
Guarding this behind a flag so we can turn it on separetly from this PR, I'll do the implementation in a follow-up.
To view it locally edit the `src/shared/features/Features.tsx` file and set `episodesFilter` to `true`